### DISCUSSION TITLE
Quest detail feature from CoP

### DISCUSSION THREAD

**Discussion by seaz5150** [OGSR/OGSR-Engine#292] [Ideas]
Hello, it would be neat if you added the feature from CoP where a long press of tab shows the details of a quest under the quest name.

Cheers, great work on the engine btw.

**Comment by seaz5150** [comment]
By the way why was the NPC death animation feature removed (and would it be possible for me to put it back for myself)?

**Comment by xrSimpAI** [comment]
> By the way why was the NPC death animation feature removed (and would it be possible for me to put it back for myself)?

This works if you do everything as stated on the wiki. ~~Crossed out~~, because it periodically works strangely and it is not recommended to enable it.

**Comment by seaz5150** [comment]
Oh thanks, I was using a wrong animation file.
